src/examples/indefinites/grammar.py

```diff
@@ -1,8 +1,16 @@
 from ultk.language.grammar import Grammar, Rule
+import pandas as pd
+from ultk.language.semantics import Universe
 
-# indefinites_grammar = Grammar.from_yaml("indefinites/grammar.yml")
-indefinites_grammar = Grammar.from_module("indefinites.grammar_functions")
+referents = pd.read_csv("indefinites/referents.csv")
+prior = pd.read_csv("indefinites/data/Beekhuizen_priors.csv")
+assert (referents["name"] == prior["name"]).all()
+referents["probability"] = prior["probability"]
+universe = Universe.from_dataframe(referents)
 
+indefinites_grammar = Grammar.from_yaml("indefinites/grammar.yml")
+#indefinites_grammar = Grammar.from_module("indefinites.grammar_functions")
+print(indefinites_grammar.parse("and(not(K+), or(N-, not(SE-)))").hm_sample(indefinites_grammar, [(universe.referents[2], True)]))
```

Comment:
It's important for the indefinites example to not modify this file directly.  The value `indefinites_grammar` is used in other scripts in the example.

Can you make an example demo-ing the MH sampling elsewhere (either its own file here, or a new sub-folder)?